cg/constants/report.py

```diff
@@ -124,6 +127,8 @@
 
 REQUIRED_SAMPLE_RNAFUSION_FIELDS: list[str] = REQUIRED_SAMPLE_BALSAMIC_FIELDS
 
+REQUIRED_SAMPLE_TOMTE_FIELDS: list[str] = REQUIRED_SAMPLE_MIP_DNA_FIELDS
```

Comment:
I think in this case it makes sense because of the nature of the analysed data (cancer vs rare disease). But it's a good point, I will instead rename them to cancer and rare disease required fields 👍